app/models/missing_model.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+class missing_model extends Model {
+
+    private function execute($sql, $datatype = null, $data = array()) {
+        $stmt = $this->DB->prepare($sql);
+        if (!$stmt) {
+            return false;
+        }
+        if (!empty($datatype) && !empty($data)) {
+            if (!call_user_func_array(array($stmt, "bind_param"), array_merge([$datatype], $data))) {
+                return false;
+            }
+        }
+        if (!$stmt->execute()) {
+            return false;
+        }
+        $result = $stmt->get_result();
```

Comment:
@nisargjhaveri Gives error, Fatal error: Call to undefined method mysqli_stmt::get_result() in /saathi/app/models/missing_model.php on line 18.
Can you check this out?